src/main/java/com/elixr/poc/rest/controller/PurchaseModificationController.java

```diff
@@ -0,0 +1,37 @@
+package com.elixr.poc.rest.controller;
+
+import com.elixr.poc.exception.IdNotFoundException;
+import com.elixr.poc.rest.request.PurchaseRequest;
+import com.elixr.poc.service.PurchaseService;
+import jakarta.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
+
+@RestController
+@Validated
+@RequestMapping("/controller")
```

Comment:
Why do you have /controller in the path? It does not match the requirement

Reply:
Removed.